Formula/a/alejandra.rb

```diff
@@ -0,0 +1,28 @@
+class Alejandra < Formula
+  desc "Command-line tool for formatting Nix Code"
+  homepage "https://kamadorueda.com/alejandra/"
+  url "https://github.com/kamadorueda/alejandra/archive/refs/tags/4.0.0.tar.gz"
+  sha256 "f3f9989c3fb6a56e2050bf5329692fae32a2b54be7c0652aa394afe4660ebb74"
+  license "Unlicense"
```

Comment:
We can add a `head` block to support installing from git with `brew install --head`:
```suggestion
  license "Unlicense"
  head "https://github.com/kamadorueda/alejandra.git", branch: "main"
```